src/lang/po/fr/Prusa-Firmware-Buddy_fr.po

```diff
@@ -76,7 +76,7 @@ msgstr "Acquisition du code d'enregistrement, veuillez patienter..."
 #. window_menu
 #: src/gui/MItem_tools.cpp:285
 msgid "A crash dump is being saved."
-msgstr "Un dump sur crash est en cours d'enregistrement."
+msgstr "Un rapport de vidage de mémoire est en cours d'enregistrement."
```

Comment:
This is not a correct translation. "rapport de vidage de mémoire" would translate to "a memory emptying report". A "crash dump" is the opposite: it's a dump of the microcontroller's (or process') memory relevant to the crash.

What would you think of using "Une copie de la mémoire vive est en cours d'enregistrement."

Reply:
ok, i will include it.